src/logging.cc

```diff
@@ -161,6 +161,8 @@ static const char* DefaultLogDir() {
   return "";
 }
 
+GLOG_DEFINE_int32(logfile_mode, 0644, "Log file mode/permissions");
```

Comment:
The default permission is different than before.

I think it was wrong before, though? The "Unix tradition" is to use 0666, and let the `umask` do its work. What do you think @shinh, @ukai?

At any rate, it should at least be 0664, like it was.